src/core/meshcop/tcat_agent.hpp

```diff
@@ -361,7 +362,8 @@ class TcatAgent : public InstanceLocator, private NonCopyable
     bool         CanProcessTlv(uint8_t aTlvType) const;
     CommandClass GetCommandClass(uint8_t aTlvType) const;
 
-    static constexpr uint16_t kJoinerUdpPort = OPENTHREAD_CONFIG_JOINER_UDP_PORT;
+    static constexpr uint16_t kJoinerUdpPort               = OPENTHREAD_CONFIG_JOINER_UDP_PORT;
+    static constexpr uint16_t kResponseWithStatusMaxLenght = 512;
```

Comment:
```suggestion
    static constexpr uint16_t kPingPayloadMaxLength = 512;
```

suggest a rename here - it's about the max length of the payload in a "Ping" message/command.
(Also swapped t/h at the end)  In general a payload may be even longer, for example when requesting a Commissioner's certificate. But for ping it's indeed limited to 512.

Reply:
fixed.